src/o365/graph/commands/teams/teams-clone.ts

```diff
@@ -0,0 +1,187 @@
+      },
+      {
+        option: '-d --description [description]',
+        description: 'The description for the new Microsoft Teams Team. Will be left blank if not specified'
+      },
+      {
+        option: '-c --classification [classification]',
+        description: 'The classification for the new Microsoft Teams Team. If not specified, will be copied from the original Microsoft Teams Team'
+      },
+      {
+        option: '-v --visibility [visibility]',
+        description: 'Specify the visibility of the new Microsoft Teams Team. Allowed values are Private|Public. If not specified, the visibility will be copied from the original Microsoft Teams Team',
+        autocomplete: ['Private', 'Public']
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+
+      if (!args.options.teamId) {
+        return 'Required parameter teamId missing';
+      }
+
+      if (!Utils.isValidGuid(args.options.teamId)) {
+        return `${args.options.teamId} is not a valid GUID`;
+      }
+
+      if (!args.options.displayName) {
+        return 'Required option displayName missing';
+      }
+
+      if (!args.options.mailNickname) {
+        return 'Required option mailNickname missing';
+      }
+
+      if (!args.options.partsToClone) {
+        return 'Required option partsToClone missing';
+      }
+
+      return true;
+    };
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(this.name).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} Before using this command, log in to the Microsoft Graph,
+    using the ${chalk.blue(commands.LOGIN)} command.
+          
+  Remarks:
+          
+    To clone a Microsoft Teams team, you have to first log in to the Microsoft
+    Graph using the ${chalk.blue(commands.LOGIN)} command, eg. ${chalk.grey(`${config.delimiter} ${commands.LOGIN}`)}.
+
+    Using this command, global admins and Microsoft Teams service admins can
+    access teams that they are not a member of.
+
+    When tabs are cloned, they are put into an unconfigured state and they are 
+    displayed on the tab bar in Microsoft Teams, and the first time you open them, 
+    you'll go through the configuration screen. (If the person opening the tab does 
+    not have permission to configure apps, they will see a message explaining that 
+    the tab hasn't been configured.)
+
+  Examples:
+    
+    Creates a copy of a Microsoft Teams team with mandatory parameters
+      ${chalk.grey(config.delimiter)} ${commands.TEAMS_CLONE} --teamId 15d7a78e-fd77-4599-97a5-dbb6372846c5 --displayName "Library Assist" --mailNickname "libassist" --partsToClone "apps,tabs,settings,channels,members" 
+    
+    Creates a copy of a Microsoft Teams team with mandatory and optional parameters
```

Comment:
Would it be more precise if we say `Create a clone ...`?

Reply:
Sure, This is complete. Please verify the latest code.